service/mcs/persistent/src/main/resources/create_cassandra_schema.cql

```diff
@@ -41,4 +41,11 @@ CREATE TABLE representation_versions (
 	creation_date timestamp,
 	PRIMARY KEY (cloud_id, schema_id, version_id)
 );
-CREATE INDEX representations_provider_id ON representation_versions (provider_id);
\ No newline at end of file
+CREATE INDEX representations_provider_id ON representation_versions (provider_id);
+
+CREATE TABLE data_sets_representations_names(

```

Comment:
What is wrong with this implementation (beside naming)? Could You write Your remarks and I will fix it (if valid).